packages/api-metadata/src/Metadata/MetadataVersioned.ts

```diff
@@ -129,70 +129,70 @@ class MetadataEnum extends Enum {
   /**
    * @description `true` if Deprecated
    */
-  public get isDeprecated (): boolean {
+  public get isDeprecated(): boolean {
     return this.type === 'MetadataDeprectated';
   }
 
   /**
    * @description `true` if V0
    */
-  public get isV0 (): boolean {
+  public get isV0(): boolean {
     return this.type === 'V0';
   }
 
   /**
    * @description `true` if V1
    */
-  public get isV1 (): boolean {
+  public get isV1(): boolean {
     return this.type === 'V1';
   }
 
   /**
    * @description `true` if V2
    */
-  public get isV2 (): boolean {
+  public get isV2(): boolean {
     return this.type === 'V2';
   }
 
   /**
    * @description `true` if V3
    */
-  public get isV3 (): boolean {
+  public get isV3(): boolean {
     return this.type === 'V3';
   }
 
   /**
    * @description `true` if V4
    */
-  public get isV4 (): boolean {
+  public get isV4(): boolean {
     return this.type === 'V4';
   }
 
   /**
    * @description `true` if V5
    */
-  public get isV5 (): boolean {
+  public get isV5(): boolean {
     return this.type === 'V5';
   }
 
   /**
    * @description `true` if V6
    */
-  public get isV6 (): boolean {
+  public get isV6(): boolean {
     return this.type === 'V6';
   }
 
   /**
    * @description `true` if V7
    */
-  public get isV7 (): boolean {
+  public get isV7(): boolean {
```

Comment:
Not sure why these keep slipping into PRs lately - this form is wrong. (Not just here and by you, but by a number of people)

Reply:
(Really seems like an auto-save thing since it mucks up whole files at once in all cases)

Reply:
My guess is: people have auto-save one, and auto-save will use the .eslintrc.js. But eslint doesn't work on my computer (I guess on others neither), because e.g. `yarn lint` gives:
```
FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory
```

same Vscode config works on smaller projects with eslint, but this one is too big. Not 100%, just a guess

Reply:
:(